mmv1/products/networkconnectivity/ServiceConnectionPolicies.yaml

```diff
@@ -0,0 +1,133 @@
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ServiceConnectionPolicy'
+base_url: 'projects/{{project}}/locations/{{location}}/serviceConnectionPolicies'
+create_url: 'projects/{{project}}/locations/{{location}}/serviceConnectionPolicies?serviceConnectionPolicyId={{name}}'
+self_link: 'projects/{{project}}/locations/{{location}}/serviceConnectionPolicies/{{name}}'
+min_version: ga
+update_verb: :PATCH
+update_mask: true
+description:
+  'Manage Service Connection Policies.'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/secure-web-proxy/docs/reference/networkconnectivity/rest/v1/projects.locations.networkConnectionPolicies'
+  guides:
+    'About Service Connection Policies': 'https://cloud.google.com/vpc/docs/about-service-connection-policies#service-policies'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+import_format:
+  ['projects/{{project}}/locations/{{location}}/serviceConnectionPolicies/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: ga
+    name: 'network_connectivity_policy_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-network-connectivity-policy'
+      consumer_network_name: 'consumer-net'
+      consumer_subnetwork_name: 'consumer-subnet'
+      producer_network_name: 'producer-net'
+      producer_subnetwork_name: 'producer-subnet'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of a ServiceConnectionPolicy. Format: projects/{project}/locations/{location}/serviceConnectionPolicies/{service_connection_policy} See: https://google.aip.dev/122#fields-representing-resource-names
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    description: |
+      The location of the tls inspection policy.
+    url_param_only: true
+properties:
```

Comment:
Some of these properties seem required, like network. I'm curious about the behavior of `location` as well, if it is not required.

Reply:
Makes sense, my bad, added the required for that.